sdk/plugin/grpc_backend_client.go

```diff
@@ -194,7 +194,6 @@ func (b *backendGRPCPluginClient) Cleanup(ctx context.Context) {
 	if server != nil {
 		server.(*grpc.Server).GracefulStop()
 	}
-	b.clientConn.Close()
```

Comment:
If this is the only reason we are stashing the connection as a member variable can we delete the variable from the backendGRPCPluginClient struct?

Reply:
Good call, removed in d2b0eb95a9c2c40b5590818a2f62c455624f33aa

Reply:
I think you'll need to remove in https://github.com/hashicorp/vault/blob/main/sdk/plugin/backend.go#L62 or it won't build.

Reply:
Whoops! Thanks 🙇🏼